src/mnode/mnode-governance.cpp

```diff
@@ -267,7 +270,7 @@ void CMasternodeGovernance::ProcessMessage(CNode* pfrom, std::string& strCommand
     if (strCommand == NetMsgType::GOVERNANCESYNC) { //Governance Payments Request Sync
     
         //TODO: Fix governance tickets processing
-        return;
+        //return; SHOULD BE FIXED BY NOW
```

Comment:
Any reason to keep these comments?   TODO && SHOULD BE FIXED??

Reply:
This is from commits on master branch after rebasing. Not belong to this PR.